File: app/src/main/java/com/example/nejat/weatherapp/POJO/WeatherResponse.java
package com.example.nejat.weatherapp.POJO;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class WeatherResponse {
    @SerializedName("dt_txt")
    private String date; // date text
    @SerializedName("main")
    private Main weatherDetail;
    private Wind wind;
    @SerializedName("weather")
    private List<Weather> weatherDesc;

    public WeatherResponse(String date, Main weatherDetail, Wind wind, List<Weather> weatherDesc) {
        this.date = date;
        this.weatherDetail = weatherDetail;
        this.wind = wind;
        this.weatherDesc = weatherDesc;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Main getWeatherDetail() {
        return weatherDetail;
    }

    public void setWeatherDetail(Main weatherDetail) {
        this.weatherDetail = weatherDetail;
    }

    public Wind getWind() {
        return wind;
    }

    public void setWind(Wind wind) {
        this.wind = wind;
    }

    public List<Weather> getWeatherDesc() {
        return weatherDesc;
    }

    public void setWeatherDesc(List<Weather> weatherDesc) {
        this.weatherDesc = weatherDesc;
    }
}

File: app/src/main/java/com/baryshev/dmitriy/daresayweather/common/data/weather/pojo/WeatherResponse.kt
package com.baryshev.dmitriy.daresayweather.common.data.weather.pojo
import com.google.gson.annotations.SerializedName


/**
 * 4/12/2018.
 */

data class WeatherResponse(
		@SerializedName("coord") val coord: Coord?,
		@SerializedName("sys") val sys: Sys?,
		@SerializedName("weather") val weather: List<Weather>?,
		@SerializedName("main") val main: Main?,
		@SerializedName("wind") val wind: Wind?,
		@SerializedName("rain") val rain: Rain?,
		@SerializedName("clouds") val clouds: Clouds?,
		@SerializedName("dt") val dt: Int?,
		@SerializedName("id") val id: Int?,
		@SerializedName("name") val name: String?,
		@SerializedName("cod") val cod: Int?
                          )









